lms/djangoapps/mobile_api/users/views.py

```diff
@@ -270,14 +270,18 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
     # the default behavior by setting the pagination_class to None.
     pagination_class = None
 
+    def is_org(self, check_org, course_org):
+        return check_org == None or (check_org.lower() == course_org.lower())
+
     def get_queryset(self):
         enrollments = self.queryset.filter(
             user__username=self.kwargs['username'],
             is_active=True
         ).order_by('created').reverse()
+        org = self.request.query_params.get('org', None)
         return [
             enrollment for enrollment in enrollments
-            if enrollment.course_overview and
+            if enrollment.course_overview and self.is_org(org, enrollment.course_overview.org) and
```

Comment:
@jcdyer Unfortunately course_overview is a class property, not a foreign key. I could look at fixing that but I think it should be in a separate PR. The course_overview prop is required by is_mobile_available_for_user anyway so my changes aren't adding extra database calls.